md/profileCreation.md

```diff
@@ -0,0 +1,79 @@
+# Profile Editor
+
+## Definition
+
+Basically, a **profile** is an XML file. It contains the profile name, the mapping with the organization (which users / groups / roles / memberships have this profile) and the portal menu entries (menus accessible to a user logged in the portal with this profile).  
+A profile must be imported onto the portal, in the **Profiles** tab, to enable the profile (you need to connect with the **Administrator** profile to do it).  
+Before Bonita 7.6, this portal tab was the only way to create and manage profiles.  
+For development purposes, the Studio can **deploy** profiles onto the portal for you (_only for Performance and Efficiency editions_).
+
+Profiles are packaged in **files of profiles**.  
+An **file of profiles** is an XML file containing one or more profiles.  
+```xml
+<!-- file of profiles -->
+<profiles>
+	<profile>
+		<!--  First Profile -->
+	</profile>
+	<profile>
+		<!--  Second Profile -->
+	</profile>
+</profiles>
+```
+
+However, profiles in a given file of profiles are totally **independent**.  
+The purpose of grouping profiles in one file is to manage together profiles related to a same business application.  
+For example, if you want to create a leave management application for four kind of users (Employees, Managers, Human Resources officers, and Administrators) then you will have to create four applications and four profiles, and to bind each profile to the related application. As for the four applications, the four profiles should be managed together, and so grouped in the same file of profile. 
+
+There is a special file of profiles for default profiles: `default_profiles.xml`  
+This file can only contain default profiles and cannot be renamed or deleted.
+
+Only Performance and Efficiency editions can create new file of profiles, in order to create custom profiles.
+
+## Create and build a profile
+
+::: info
+**Note:** All Bonita editions come with an .xml editor for this feature. Only Performance and Efficiency editions also come with a graphical UI.
+:::
+
+From Bonita Studio, click on **Organization** -> **Profile** -> **New...**  
```

Comment:
This should be "Profiles", even in the product, like "Connectors".
"REST API extensions" should also be made plural, like "Application descriptors" in the Development menu.